hikari/guilds.py

```diff
@@ -1220,6 +1220,61 @@ def discovery_splash_url(self) -> typing.Optional[files.URL]:
         """Discovery splash URL for the guild, if set."""
         return self.make_discovery_splash_url()
 
+    @property
+    def members(self) -> typing.Mapping[snowflakes.Snowflake, Member]:
```

Comment:
Does the test layout need to be updated to reflect these changes?

Reply:
Yep, will do that when I can. I meant to open this as a draft :sweat_smile:

Reply:
Moved tests :+1: